Base/QTCore/CMakeLists.txt

```diff
@@ -199,6 +199,12 @@ if(Slicer_BUILD_DICOM_SUPPORT)
     ${DCMTK_LIBRARIES}
     CTKDICOMCore
     )
+  foreach(lib IN LISTS DCMTK_LIBRARIES)
+    if(TARGET DCMTK::${lib})
+      list(REMOVE_ITEM KIT_target_libraries ${lib})
+      list(APPEND KIT_target_libraries DCMTK::${lib})
+    endif()
+  endforeach()
```

Comment:
@jcfr These type of changes in Slicer, CTK and ITK appear to be compatible with the change in https://github.com/DCMTK/dcmtk/commit/c684bb333e9fe43f9e99de2d4398eed48192540b. DCMTK 3.6.7 is building/packaging/testing successfully on the Windows platform. On this topic of namespace targets (https://cmake.org/cmake/help/latest/manual/cmake-developer.7.html?highlight=namespace) I would assume there is probably a simpler/better solution, though what I have currently seems to work. Let me know with your CMake expertise if we should proceed with these changes or do something else. These changes are similar to https://github.com/commontk/CTK/pull/1053 where MITK used that branch to add DCMTK 3.6.7 support.

I'm not sure if just an improvement needs to be made to the FindDCMTK of CMake (https://gitlab.kitware.com/cmake/cmake/-/blob/master/Modules/FindDCMTK.cmake) with the DCMTK_LIBRARIES entry being compatible with the appropriate new DCMTK namespace, but then again I guess we wouldn't want to necessarily require/rely on developers using a new CMake version that includes this potential upstream update.

Reply:
There is also a "catchall" target in exported DCMTK targets, named DCMTK::DCMTK which would links available in the referenced DCMTK build. Maybe this is also a considerable option.